engine/libstats_test.go

```diff
@@ -730,6 +730,10 @@ func (sMM *statMetricMock) AddEvent(evID string, ev utils.DataProvider) error {
 	return nil
 }
 
+func (sMM *statMetricMock) OneEvent(ev utils.DataProvider) error {
+	return nil
+}
+
 func (sMM *statMetricMock) RemEvent(evTenantID string) error {
 	switch sMM.testcase {
 	case "remExpired error":
```

Comment:
You would need to modify all the previous tests and also have close to 100% in statmetrics_tests.go before we can accept the PR